index.js

```diff
@@ -87,6 +118,14 @@ async function getResponse(request, config) {
  * This function will create a Request object based on configuration's axios
  */
 function createRequest(config) {
+    // Fix bug https://stackoverflow.com/questions/39280438/fetch-missing-boundary-in-multipart-form-data-post
+    if (config.data instanceof FormData && config.headers) {
+        const ContentType = 'Content-Type';
+        delete config.headers[ContentType];
+        delete config.headers[ContentType.toLowerCase()];
+        delete config.headers[ContentType.toUpperCase()];
+    }
+
```

Comment:
We don't need this. There's code to do it later in the code.  Your use case seems similar to mine, which is run axios on a ServiceWorker. I think we just need to fix the condition in the code a little bit below to make sure it enters there. It's not entering the `if` condition in case of ServiceWorkers because `isStandardBrowserEnv` is returning `false` as ServiceWorkers don't have access to `window` nor `document`.